### PR TITLE
[batch] rename batch2 => batch

### PR DESCRIPTION
**Pull request by cseed**
(probably partial) list of things we'll have to do manually once this goes in:

 - rename account batch2 => batch (and gsa-key, tokens, etc.)
 - rebuild database
 - had deploy (which includes cleaning up old logs and bucket)
 - batch service account, roles and role bindings

### CODE REVIEW

File: batch/Makefile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	docker build -t batch-worker -f Dockerfile.worker --cache-from batch-worker,$(BATCH_WORKER_LATEST),python:3.6-slim-stretch ..
+
+push: build
+	docker tag batch $(BATCH_LATEST)
+	docker push $(BATCH_LATEST)
+	docker tag batch $(BATCH_IMAGE)
+	docker push $(BATCH_IMAGE)
+	docker tag batch $(BATCH_WORKER_LATEST)
+	docker push $(BATCH_WORKER_LATEST)
+	docker tag batch $(BATCH_WORKER_IMAGE)
```
Comment (jigold) [inline batch/Makefile]: shouldn't these be batch-worker?

File: batch/create-batch-tables-pod.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 apiVersion: v1
```
Comment (jigold) [inline batch/create-batch-tables-pod.yaml]: I can't make a comment above. I think you need to change the name of the file `build-batch2-worker-image-startup.sh`

File: batch/deployment.yaml [hunk 3]
```diff
@@ -85,32 +85,32 @@ spec:
...
          secret:
            optional: false
-           secretName: "{{ batch2_database.user_secret_name }}"
+           secretName: "{{ batch_database.user_secret_name }}"
        - name: gsa-key
          secret:
            optional: false
-           secretName: batch2-gsa-key
-       - name: batch2-tokens
+           secretName: batch-gsa-key
```
Comment (jigold) [inline batch/deployment.yaml]: Does the secrets batch-tokens and batch-gsa-key already exist?

File: build.yaml [hunk 4]
```diff
@@ -208,15 +208,15 @@ steps:
          kubectl -n {{ default_ns.name }} create secret generic batch-tokens --from-file=/user-tokens/tokens.json
      fi
      # created batch
-     # create batch2
-     N=$(kubectl -n {{ default_ns.name }} get secret --ignore-not-found=true --no-headers batch2-tokens | wc -l | tr -d '[:space:]')
+     # create batch
+     N=$(kubectl -n {{ default_ns.name }} get secret --ignore-not-found=true --no-headers batch-tokens | wc -l | tr -d '[:space:]')
```
Comment (jigold) [inline build.yaml]: There's still the block above creating the original batch keys and tokens.

File: batch/Makefile (hunk 1 above)
Comment (cseed) [reply batch/Makefile]: Isn't it?

 > BATCH_WORKER_IMAGE = gcr.io/$(PROJECT)/batch-worker:...

Maybe I don't understand the question.

Comment (jigold) [reply batch/Makefile]: I thought the tags should be batch-worker.

Comment (cseed) [reply batch/Makefile]: Oops, 100%, good catch!

File: batch/batch/driver/instance_pool.py [hunk 5]
```diff
@@ -277,7 +277,7 @@ def add_instance(self, instance):
 NAME=$(curl -s http://metadata.google.internal/computeMetadata/v1/instance/name -H 'Metadata-Flavor: Google')
 
 # this has to match LogStore.worker_log_path
-gsutil -m cp run.log worker.log /var/log/syslog gs://$BUCKET_NAME/batch2/logs/$INSTANCE_ID/worker/$NAME/
+gsutil -m cp run.log worker.log /var/log/syslog gs://$BUCKET_NAME/batch/logs/$INSTANCE_ID/worker/$NAME/
```
Comment (jigold) [inline batch/batch/driver/instance_pool.py]: I can't make the comment in the right place. `batch2-agent` tag needs to be changed to match the build step `delete-batch_instances`

**Pull request merged**